### PR TITLE
Added URL based navigation

### PR DESCRIPTION
**Pull request by DerProfi**

### CODE REVIEW

**Comment by DerProfi** [comment]
Hey @viniciuskneves, this is my url based navigation approach. If you think it's good, I'll merge this one before adjusting the test in "jest" branch, which tests the router push.

**Review by viniciuskneves — COMMENTED** [review]
Really cool this one. I recommend you to take a look into https://router.vuejs.org/guide/essentials/passing-props.html#passing-props-to-route-components and forwarding the param to your view directly.

**Comment by DerProfi** [comment]
hey @viniciuskneves , changed the way the data is being passed. I would now merge and adjust the test without mocking the route.

File: src/router/index.js [hunk 1]
```diff
@@ -10,11 +10,13 @@ const routes = [
     path: "/",
     name: "Form",
     component: Form,
+    props: true,
```
Comment (viniciuskneves) [inline src/router/index.js:13]: I don't think you need `props` here. You can keep it only where it is needed.

Comment (viniciuskneves) [reply src/router/index.js:13]:
```suggestion
```

**Review by viniciuskneves — COMMENTED** [review]
Really cool changes! I just want to warn you that after merging it, your tests will change a bit (I think they will becomes simpler at some extent).

File: src/router/index.js (hunk 1 above)
Comment (DerProfi) [reply src/router/index.js:13]: Thank you. Thought I also have to add it to the "sender"

**Pull request merged**